src/flownet/network_model/_mitchell.py

```diff
@@ -12,7 +12,8 @@ def mitchell_best_candidate_modified_3d(
     perforations: List[Coordinate],
     num_added_flow_nodes: int,
     num_candidates: int = 1000,
-    hull_factor: Optional[float] = None,
+    place_nodes_in_volume_reservoir: Optional[bool] = None,
+    hull_factor: float = 1.2,
```

Comment:
Why 1.2? Previously it was `None`.

Reply:
@wouterjdb : True, but it is defaulted in the config parser to 1.2, similarly as `num_candidates` is also defaulted to 1000. So that's why I aligned the notation, as it was never `None` anyway. But maybe as they always have a value anyway, we should just not re-assign them here at all (like `num_add_flow_nodes`)